Libplanet.Tests/Blockchain/BlockChainTest.Append.cs

```diff
@@ -728,5 +731,75 @@ public void CannotAppendBlockWithInvalidActions()
                 () => _blockChain.Append(block, TestUtils.CreateBlockCommit(block)));
             Assert.Equal(0, _blockChain.Tip.Index);
         }
+
+        [SkippableFact]
+        public void MigrateStateWithoutAction()
+        {
+            var policy = new BlockPolicy(
+                blockAction: new MinerReward(1),
+                getMaxTransactionsBytes: _ => 50 * 1024);
+            var stagePolicy = new VolatileStagePolicy();
+            var fx = GetStoreFixture(policy.BlockAction);
+            var renderer = new ValidatingActionRenderer();
+            var actionEvaluator = new ActionEvaluator(
+                _ => policy.BlockAction,
+                stateStore: fx.StateStore,
+                actionTypeLoader: new SingleActionLoader(typeof(DumbAction)));
+
+            var txs = new[]
+            {
+                Transaction.Create(
+                    0,
+                    fx.Proposer,
+                    null,
+                    actions: new IAction[]
+                    {
+                        new Initialize(
+                            validatorSet: TestUtils.ValidatorSet,
+                            states: ImmutableDictionary.Create<Address, IValue>()),
+                    }.ToPlainValues(),
+                    timestamp: DateTimeOffset.UtcNow),
+            };
+            PreEvaluationBlock preEvalGenesis = new BlockContent(
+                new BlockMetadata(
+                    protocolVersion: BlockMetadata.LegacyStateVersion,
+                    index: 0L,
+                    timestamp: DateTimeOffset.UtcNow,
+                    miner: fx.Proposer.ToAddress(),
+                    publicKey: fx.Proposer.PublicKey,
+                    previousHash: null,
+                    txHash: BlockContent.DeriveTxHash(txs),
+                    lastCommit: null),
+                transactions: txs).Propose();
+            var genesis = preEvalGenesis.Sign(
+                fx.Proposer,
+                BlockChain.DetermineGenesisStateRootHash(actionEvaluator, preEvalGenesis, out _));
+            var blockChain = BlockChain.Create(
+                policy,
+                stagePolicy,
+                fx.Store,
+                fx.StateStore,
+                genesis,
+                actionEvaluator,
+                renderers: new[] { new LoggedActionRenderer(renderer, Log.Logger) });
+            Assert.True(blockChain.GetWorldState().Legacy);
+            var emptyBlockContent = new BlockContent(
+                new BlockMetadata(
+                    protocolVersion: BlockMetadata.LegacyStateVersion,
+                    index: 0L,
+                    timestamp: DateTimeOffset.UtcNow,
+                    miner: fx.Proposer.ToAddress(),
+                    publicKey: fx.Proposer.PublicKey,
+                    previousHash: null,
+                    txHash: BlockContent.DeriveTxHash(txs),
+                    lastCommit: null),
+                transactions: txs);
```

Comment:
Seems like this variable doen't used at all.
May be, test appending `Block` with `LegacyStateVersion` after genesis is absent?